tests/system/python/packages/test_filters.py

```diff
@@ -0,0 +1,710 @@
+# FOGLAMP_BEGIN
+# See: http://foglamp.readthedocs.io/
+# FOGLAMP_END
+
+""" Test Filters Package System tests:
+        foglamp-south-http-south south plugin
+        foglamp-filter-expression, foglamp-filter-python35, foglamp-filter-simple-python, foglamp-filter-ema filter plugins
+"""
+
+__author__ = "Yash Tatkondawar"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems Inc."
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+import subprocess
+import http.client
+import json
+import pytest
+import os
+import time
+import urllib.parse
+from pathlib import Path
+
+PROJECT_ROOT = ""
+SCRIPTS_DIR_ROOT = ""
+ASSET_NAME_PY35 = "end_to_end_py35"
+ASSET_NAME_SP = "end_to_end_sp"
+ASSET_NAME_EMA = "end_to_end_ema"
+TEMPLATE_NAME = "template.json"
+SENSOR_VALUE = 20
+package_build_version = "nightly"
+
+
+def set_project_root():
+    global PROJECT_ROOT, SCRIPTS_DIR_ROOT
+    PROJECT_ROOT = Path(__file__).parent.parent.parent.parent.parent
+    SCRIPTS_DIR_ROOT = "{}/tests/system/python/packages/data/".format(PROJECT_ROOT)
+
+
+def post_request(foglamp_url, post_url, payload):
+    conn = http.client.HTTPConnection(foglamp_url)
+    conn.request("POST", post_url, json.dumps(payload))
+    res = conn.getresponse()
+    assert 200 == res.status
+    res = res.read().decode()
+    r = json.loads(res)
+    return r
+
+
+def get_request(foglamp_url, get_url):
+    con = http.client.HTTPConnection(foglamp_url)
+    con.request("GET", get_url)
+    resp = con.getresponse()
+    r = json.loads(resp.read().decode())
+    return r
+
+
+def put_request(foglamp_url, put_url, payload):
+    conn = http.client.HTTPConnection(foglamp_url)
+    conn.request("PUT", put_url, json.dumps(payload))
+    res = conn.getresponse()
+    assert 200 == res.status
```

Comment:
Usually we do not add assertions for the generic functions.
But it would be worth to add `assert 200 == res.status, "ERROR! PUT {} request failed".format(put_url)`